dags/openshift_nightlies/config/install/aws/acs.json

```diff
@@ -7,26 +7,26 @@
     "openshift_host_prefix": "22",
     "openshift_network_type": "OpenShiftSDN",
     "openshift_master_instance_type": "r5.4xlarge",
-    "openshift_worker_instance_type": "m5.2xlarge",
+    "openshift_worker_instance_type": "m6i.2xlarge",
     "openshift_master_root_volume_size": 220,
-    "openshift_master_root_volume_type": "io1",
+    "openshift_master_root_volume_type": "gp3",
     "openshift_master_root_volume_iops": 3000,
     "openshift_worker_root_volume_size": 100,
-    "openshift_worker_root_volume_type": "gp2",
+    "openshift_worker_root_volume_type": "gp3",
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_infra_node_instance_type": "m5.4xlarge",
-    "openshift_workload_node_instance_type": "m5.4xlarge",
+    "openshift_infra_node_instance_type": "m6i.4xlarge",
+    "openshift_workload_node_instance_type": "m6i.4xlarge",
     "openshift_infra_node_volume_size": 100,
-    "openshift_infra_node_volume_type": "gp2",
+    "openshift_infra_node_volume_type": "gp3",
     "openshift_infra_node_volume_iops": 0,
     "openshift_workload_node_volume_size": 500,
-    "openshift_workload_node_volume_type": "gp2",
+    "openshift_workload_node_volume_type": "gp3",
     "openshift_workload_node_volume_iops": 0,
     "openshift_prometheus_retention_period": "15d",
-    "openshift_prometheus_storage_class": "gp2",
+    "openshift_prometheus_storage_class": "gp3-csi",
```

Comment:
Can we use the original gp2 storageclass to ensure this doesn't break 4.9 installations?

Reply:
@amitsagtani97

Reply:
Updated :+1: